README.md

```diff
@@ -145,7 +145,7 @@ please see the [exporter README.md](exporter/exporterparser/README.md).
 ```yaml
 exporters:
   opencensus:
-    endpoint: "http://localhost:10001"
+    endpoint: "localhost:10001"
```

Comment:
:+1: valid syntax is defined here https://github.com/grpc/grpc/blob/master/doc/naming.md